README.md

```diff
@@ -150,8 +153,8 @@ username                         | Text |
 The data in the 'commcare_users' table comes from the [List Mobile Workers
 API endpoint](https://confluence.dimagi.com/display/commcarepublic/List+Mobile+Workers).
 
-Specifying the --locations option will export an additional table named
-'commcare_locations' containing the following columns:
+Specifying the --locations option or --with-organization options will export
+an additional table named 'commcare_locations' containing the following columns:
 
 Column                       | Type | Note
 ------                       | ---- | ----
```

Comment:
Looks like `parent` column is missing here. Was that intentionally left out? Since the column has been around for a while, maybe it makes sense to simply document that the value of `parent` is a resource URI rather than changing the column name as in https://github.com/dimagi/commcare-export/pull/138/commits/6a7374c1f8cb1185b13c6833b15f1ca6d95487c5?

Reply:
OK, I'll add parent with a coment.